gradio/examples.py

```diff
@@ -290,7 +290,7 @@ async def cache(self) -> None:
                 if self.batch:
                     processed_input = [[value] for value in processed_input]
                 prediction = await Context.root_block.process_api(
-                    fn_index, processed_input
+                    fn_index=fn_index, inputs=processed_input, request=None
```

Comment:
Might be worth mentioning that example caching may raise an exception if the function takes in a request.

Reply:
Hmm let me test this out

Reply:
Yeah ok so the value of `request` will be `None`. I'll add to the Guide that you should handle the case when `request` is None.